resources/build/offline-help-style-spec.txt

```diff
@@ -0,0 +1,258 @@
+<style>
+/**************************************
+ * CSS RESET
+ * ***********************************/
+
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed,
+figure, figcaption, footer, header, hgroup,
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+  padding-left: 10px;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+}
+
+tbody {
+    display: table-row-group;
+    vertical-align: middle;
+}
+
+table td{
+	padding: 4px;
+}
+
+html{
+		font-family: 'Cabin', sans-serif;
+		color: #333;
+}
+
+html p,
+html dd,
+html li {
+  font-family: 'Calibri', sans-serif;
+  font-size: 14pt;
+}
+
+body {
+	line-height: inherit;
+}
+
+/**************************************
+ * Page Layout CSS
+ * ***********************************/
+
+html{
+    cursor: default;
+    background: #fff;
+}
+
+:not(div.interface-hierarchy) blockquote {
+  margin-left: 32px;
+  margin-bottom: 16px;
+  background: rgba(242,242,242,0.9);
+  padding: 4px;
+  border-radius: 4px;
+}
+
+h1{
+	text-align: left;
+	font-size: 28pt;
+	padding: 40px 10px 10px 10px;
+}
+
+h2{
+	font-size: 20pt;
+	line-height: 1.4;
+	padding: 32px 10px 10px 10px;
+}
+
+h3{
+	font-size: 16pt;
+	font-weight: 600;
+	padding: 16px 10px 10px 10px;
+}
+
+h3, h2, h1 {
+  color: #B92034;
+  margin-top: 10px;
+}
+
+h1 a, h2 a, h3 a {
+  color: #B92034;
+}
+
+h1 a:visited, h2 a:visited, h3 a:visited {
+  color: #B92034;
+}
```

Comment:
I added this part in manually so that header links still keep their header formatting.  I could understand if we wanted the color to vary depending upon link state (visited, hover, etc), so I thought I'd highlight the relevant spot with this comment.